packages/serverpod_shared/lib/src/config.dart

```diff
@@ -449,6 +510,15 @@ Map? _redisConfigMap(Map configMap, Map<String, String> environment) {
   ]);
 }
 
+Map? _sessionLogsConfigMap(Map configMap, Map<String, String> environment) {
```

Comment:
Suggestion: Add `build` to the beginning to be consistent
```suggestion
Map? _buildSessionLogsConfigMap(Map configMap, Map<String, String> environment) {
```

Reply:
@hampuslavin if i do this for this one then we should do for the others, what do u think ?
`_databaseConfigMap`
`_redisConfigMap`
ect ect